pkgs/development/tools/prettierd/default.nix

```
@@ -0,0 +1,45 @@
+{ lib
+, mkYarnPackage
+, fetchFromGitHub
+, nodejs
+, makeWrapper
+, extraFlags ? [ ]
+, ...
+}:
+mkYarnPackage rec {
+  pname = "prettierd";
+  version = "0.23.4";
+
+  src = fetchFromGitHub {
+    owner = "fsouza";
+    repo = "prettierd";
+    rev = "v${version}";
+    hash = "sha256-GTukjkA/53N9ICdfCJr5HAqhdL5T0pth6zAk8Fu/cis=";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  buildPhase = ''
+    runHook preBuild
+    yarn prepare
+    yarn build
+    runHook postBuild
+  '';
+
+  postInstall = ''
+    wrapProgram "$out/bin/prettierd" --prefix PATH : "${nodejs}/bin" \
+      ${lib.concatStringsSep "\n" (map (a: "--add-flags ${builtins.toJSON a}") extraFlags)}
+  '';
+
+  doDist = false;
+
+  meta = with lib; {
+    description = "Prettier, as a daemon, for improved formatting speed";
+    homepage = "https://github.com/fsouza/prettierd";
+    license = licenses.isc;
+    changelog = "https://github.com/fsouza/prettierd/blob/main/CHANGELOG.md";
+    mainProgram = "prettierd";
+    platforms = lib.platforms.linux;
```

You have a `with lib;` in that scope, thus you can omit `lib` here:

```suggestion
    platforms = platforms.linux;
```